### ISSUE TITLE
use of clmtools

### ISSUE THREAD

**Issue by saadrawasia** [auduno/clmtools#9]
I want to use clmtracker for my project. the problem is that how i can fix models on face. You have suggested clmtools for it but the problem is that i have no experience in python nor i can understand it's document. Is there any other way to make image appear on face.

If you can help me it will be of great help.

**Comment by auduno** [comment]
I assume you're meaning that you want to put new images on the face like in the face substitution demo? If so you need to get the points of the face in the image you're using. You don't need to know python to do this. You can use the annotator webpage found here : https://github.com/auduno/clmtools/tree/master/annotater

Insert the parameters/points you get from this annotator together with your image into the face substitution demo (https://auduno.github.io/clmtrackr/examples/facesubstitution.html), and it should work.

**Comment by saadrawasia** [comment]
What if i make 3d model on 3d maya or 3d max, will this annotater file be used to put model on the face?

**Comment by NomanYaqoob** [comment]
Hello there.. same issue with me... can you please suggest how can I put image like glasses or hat dynamically

**Comment by auduno** [comment]
Using a 3d model is a bit more complicated, and there is no possibility to do that in clmtrackr currently, so you'd have to figure out some way to do that on your own. I guess the 3d model could be rendered in a webGL canvas and placed relatively to face based on face position somehow, but I haven't tried it.

**Comment by saadrawasia** [comment]
Thanks

**Comment by Greggydev** [comment]
Having some trouble creating an annotated new model on the basis that a lot of the code/libraries seem deprecated, the images database required to compile a new annotated image doesn't appear to be there and some other issues... can you confirm that this project works based on the documentation you have provided ?

**Comment by znap** [comment]
Anyone have problems "uploading" more of 5 masks ? because i can't make
works with more of 5 masks, please anyone help me. thanks.

2016-11-07 15:58 GMT-03:00 Greggydev notifications@github.com:

**Comment by Greggydev** [comment]
are you able to get the annotator or model builder working ????

**Comment by znap** [comment]
Yes, i can choose my masks, i used the annotator and model builder to make
them but i only can use 5 masks, when i try add one more mask the face
tracking stop working fine.

2016-11-07 20:28 GMT-03:00 Greggydev notifications@github.com:

**Comment by Greggydev** [comment]
Can you upload your code somewhere ? I can't get this project working at all .

**Comment by znap** [comment]
Audun, it's possible work with more of 5 masks ? @auduno , please help me... Because i can't... If i add another mask face the sustitution doesn't works.
